core/packages/contracts/foundry.toml

```diff
@@ -0,0 +1,16 @@
+[profile.default]
+solc = '0.8.9'
+via_ir = false
+src = 'contracts'
+out = 'out'
+libs = ["lib","node_modules"]
+test = 'contracts/test'
+fs_permissions = [{ access = "read", path = "./"}]
```

Comment:
Didn't spot this in the config reference. Is there documentation for it?

Reply:
should require it when reading or writing file from vm 
https://book.getfoundry.sh/cheatcodes/fs
previously I use this to load external file which contains secrect key but change to env file instead later, so not nessessary also